src/applications/static-pages/createAdditionalInfoWidget.js

```diff
@@ -11,15 +12,17 @@ export default function createAdditionalInfoWidget() {
         (titleNode && titleNode.textContent) || 'More information';
       const contentContainer = qS(el, '.additional-info-content').parentNode;
       const contentMarkup = qS(el, '.additional-info-content').innerHTML;
-      const expandedContentId = `${contentContainer.id}-content`;
+      const additionalInfoId = uniqueId('additional-info-');
+      const analyticsEvent =
+        contentContainer.dataset && contentContainer.dataset.analytics;
```

Comment:
I didn't like how HTML was required to have an ID for this widget to function, so I started generating them.